### DISCUSSION TITLE
Can render map style on Globe view ?

### DISCUSSION THREAD

**Discussion by istick15** [visgl/deck.gl#6720] [Q&A]
If i have some custom style like [json style](https://demotiles.maplibre.org/style.json ) 
<img width="1586" alt="Screen Shot 2565-02-28 at 23 59 48" src="https://user-images.githubusercontent.com/52554439/156025443-6c57f31a-ba20-4d5f-a191-075e67f26553.png">

can i  render style on globe view.

my layers :

>  const backgroundLayers = useMemo(
    () => [
      new SimpleMeshLayer({
        id: "earth-sphere",
        data: [0],
        mesh: new SphereGeometry({
          radius: EARTH_RADIUS_METERS,
          nlat: 18,
          nlong: 36,
        }),
        coordinateSystem: COORDINATE_SYSTEM.CARTESIAN,
        getPosition: [0, 0, 0],
        getColor: [255, 255, 255],
      }),
      new MVTLayer({
        binary: false,
        data: https://localhost:3000/core/api/tiles/1.0-beta/tiles/public/osm,
        minZoom: 0,
        maxZoom: 23,
        lineWidthMinPixels: 1,
      }),
    ],
    []
  );

render globe
> <DeckGL
>         views={new _GlobeView()}
>         initialViewState={{
>           longitude: coor.lng,
>           latitude: coor.lat,
>           zoom: zoom,
>           maxZoom: 11,
>         }}
>         controller={true}
>         effects={[lightingEffect]}
>         layers={[backgroundLayers]}
>       ></DeckGL>

**Comment by Pessimistress** [comment]
deck.gl does not support Mapbox styles.

**Reply by istick15** [reply]
thanks for answer.
